fury/ui.py

```diff
@@ -5240,3 +5240,165 @@ def _set_position(self, coords):
         pass
         # self.actor.SetPosition(*coords)
         # self.container.SetPosition(*coords)
+
+
+class Card2D(UI):
+
+    """Card element to show image and related text
+    """
+
+    def __init__(
+        self,
+        image_path,
+        body,
+        title,
+        padding=2,
+        position=(0, 0),
+        size=(400, 400),
+        image_scale=0.5,
+        text_color=(0., 0., 0.),
+            ):
```

Comment:
* Its better if the arguments are formatted horizontally instead. (As far as PEP8 allows you to)
* Add default values for `body` and `title`.
* Add additional arguments to:
    *  make the component draggable (or not) 
    * add color/opacity to the bg panel
    * Add color to body and title separately